### ISSUE TITLE
Variable explorer issue

### ISSUE THREAD

**Issue by ShashankRaoM** [spyder-ide/spyder#9038]
### What steps will reproduce the problem?

Clicking on a varible caused this error to occur.
Each time i click any varible though it is 1 or 2d this error occurs and the table isnt shown 

### Traceback
```python-traceback
  File "C:\Users\raomsh\AppData\Local\conda\conda\envs\tensorflow\lib\site-packages\spyder\widgets\variableexplorer\collectionseditor.py", line 476, in createEditor
    if not editor.setup_and_check(value, title=key, readonly=readonly):
  File "C:\Users\raomsh\AppData\Local\conda\conda\envs\tensorflow\lib\site-packages\spyder\widgets\variableexplorer\arrayeditor.py", line 620, in setup_and_check
    self.data.flags.writeable = True
ValueError: cannot set WRITEABLE flag to True of this array
WARNING:spyder.widgets.github.backend:failed to send bug report on github. response={'code': 401, 'json': {'message': 'Bad credentials', 'documentation_url': 'https://developer.github.com/v3'}}
```

## Versions

* Spyder version: 3.3.1 
* Python version: 3.6.7
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```
pyflakes >=0.6.0  :  2.1.1 (OK)
pycodestyle >=2.3 :  2.5.0 (OK)
pygments >=2.0    :  2.3.1 (OK)
sphinx >=0.6.6    :  1.8.5 (OK)
rope >=0.9.4      :  0.12.0 (OK)
jedi >=0.9.0      :  0.13.3 (OK)
nbconvert >=4.0   :  5.4.1 (OK)
pandas >=0.13.1   :  0.24.2 (OK)
numpy >=1.7       :  1.16.2 (OK)
sympy >=0.7.3     :  None (NOK)
cython >=0.21     :  None (NOK)
qtconsole >=4.2.0 :  4.4.3 (OK)
IPython >=4.0     :  7.4.0 (OK)
matplotlib >=2.0.0:  3.0.3 (OK)
pylint >=0.25     :  2.3.1 (OK)
```

**Comment by ccordoba12** [comment]
This is fixed in our latest version (3.3.3). Please update by opening the Anaconda Prompt and running there

    conda update spyder

**Comment by ShashankRaoM** [comment]
Updating Spyder causes python to crash immediately.
